projects/python-client/src/datapane/client/api/user.py

```diff
@@ -35,22 +35,19 @@
 from .. import DPError
 from .. import config as c
 from ..analytics import capture, capture_event
-from ..utils import display_msg, success_msg
+from ..utils import display_msg, failure_msg, success_msg
 from .common import _process_res
 
 __all__ = ["login", "logout", "ping", "hello_world", "template"]
 
 
-def login(
-    token: t.Optional[str], server: str = c.DEFAULT_SERVER, env: str = c.DEFAULT_ENV, cli_login: bool = True
-) -> str:
+def login(token: t.Optional[str], server: str = c.DEFAULT_SERVER, cli_login: bool = True) -> str:
     """
     Login to the specified Datapane Server, storing the token within a config-file called `env` for future use
```

Comment:
First line of docstring here needs updating.

The `login` function is public and documented: https://datapane.github.io/datapane/#datapane.client.api.login
What's the backwards compat story here? If it's redundant, can we accept the argument and warn the user about it rather than crashing?

Reply:
Ah thanks for spotting that - hadn't twigged they were public. Do we have a policy for this sort of thing? If not it would probably make sense to raise a deprecated warning if they're supplied, then remove it completely in the next/later release?

Reply:
It's redundant, we can keep the param and ignore for now along with a deprecation warning pending removing fully in the next major bump if needed